components/sup/src/manager/service.rs

```diff
@@ -921,11 +934,23 @@ impl Service {
         win_perm::harden_path(path.as_ref())
     }
 
+    /// Called when the Supervisor reattaches itself to an already
+    /// running service. Use this to re-initiate any associated
+    /// processes, futures, etc.
+    ///
+    /// This should generally be the opposite of `Service::detatch`.
```

Comment:
Putting this definition closer to that of `detach` would probably decrease the likelihood of one being changed without complementary changes too the other.

Reply:
good call; will do